keyboards/fc980c/keymaps/via/config.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2017 Balz Guenat
+ * Via support added 2021 (c) John Graham
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+
+// place overrides here
+#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```

Comment:
Should probably go in the keyboard-level config.h so that it affects everyone who enables VIA on their own keymaps.

Reply:
This change has been made. Validated that the firmware still compiles and functions properly.